src/styles/components/_override.buttons.scss

```diff
@@ -87,6 +87,15 @@
       }
     }
   }
+
+  // Change buttons to full-width on mobile devices
+  &.mds-u-full-width-mobile {
```

Comment:
@babsmiles Maybe we should name this `mds-c-button--mobile-full-width` or something along those lines to follow the naming pattern for other button classes (e.g. `ds-c-button--inverse`)?